.azure/scripts/wheels.yml

```diff
@@ -2,37 +2,38 @@
 steps:
 - script: |
     mkdir -p dist
-    python -m pip install --upgrade pip
+    python -m pip install --upgrade pip setuptools -r test-requirements.txt
```

Comment:
The installation of `setuptools` shouldn't be important (it is declared as a build dependency)

Reply:
But it definitely is. Because we later on invoke setup.py directly. This will then NOT be executed in the build environment created by pip, but rather in the plain host environment, lacking setuptools for <=Python 3.12

Reply:
Ah yes, I see `python setup.py test_java` :+1: